hack/run_test

```diff
@@ -308,21 +295,29 @@ function run_replication_test() {
       sleep 1
     done
   done
+}
 
+function test_value_replication() {
+  local table_name="$1"; shift
+  local max_attempts=30
+
+  # Setup the replication data
   local value
   value=24
-  # Do some real work to test replication in practice
-  postgresql_cmd -c "CREATE TABLE t1 (a integer); INSERT INTO t1 VALUES ($value);"
+  local table_name="t1"
+  postgresql_cmd -c "CREATE TABLE $table_name (a integer); INSERT INTO $table_name VALUES ($value);"
 
   # Read value from slaves and check whether it is expected
   local f
   local slave_ip
+  local cidfile
+  local master_ip=$(get_container_ip master-$cid_suffix.cid)
   for f in `find ${CIDFILE_DIR} -type f -name "slave-*"`; do
```

Comment:
Here is probably missing "suffix" in find command.  Btw., I would much rather see the `run_slave` method "appending" the IP of newly created slave into let's say $slave_addresses variable.  Then you can simply iterate like `for i slave in $slave_addresses`.